staging/src/k8s.io/apiserver/pkg/endpoints/handlers/response.go

```diff
@@ -38,10 +39,109 @@ import (
 	"k8s.io/apiserver/pkg/endpoints/handlers/responsewriters"
 	"k8s.io/apiserver/pkg/endpoints/metrics"
 	endpointsrequest "k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/apiserver/pkg/storage"
 
-	klog "k8s.io/klog/v2"
+	"k8s.io/klog/v2"
 )
 
+// watchListEncoder performs encoding of
+// a special watchList bookmark event
+//
+// see: https://github.com/kubernetes/kubernetes/pull/127319
```

Comment:
Let's not use the PR number - let's rather copy the relevant parts of the comment from there.